metadata-ingestion/docs/sources/mysql/mysql_recipe.yml

```diff
@@ -9,5 +9,12 @@ source:
     username: root
     password: example
 
+    # If you need to use SSL with MySQL:
```

Comment:
I think this is more general than just MySQL.
Maybe we can create a dedicated page that explains how to configure SSL connection parameters?

Reply:
Unfortunately the `connect_args` piece is db driver specific, so it doesn't generalize very well - we'll have to go through the sources one by one to add SSL examples

Reply:
Got it. Thank you!